jep/7/README.adoc

```diff
@@ -157,5 +307,12 @@ n/a
 
 == References
 
-https://groups.google.com/d/msg/jenkinsci-dev/Ve0fqAud3Mk/MTIxw6ZyBwAJ
+* https://groups.google.com/d/msg/jenkinsci-dev/Ve0fqAud3Mk/MTIxw6ZyBwAJ
+* https://github.com/jenkinsci/ruby-runtime-plugin[Ruby Runtime Plugin]
+* https://github.com/jenkins-infra/jenkins.io/pull/4256[Announcement Draft]
+* https://github.com/jenkins-infra/jenkins.io/pull/4256[Announcement blogpost]
+
+== Change history
 
+* April 2021 - Extend the scope to include the Python plugin runtime,
+  clarify the scope and the rollout plan based on the dev list discussion.
```

Comment:
```suggestion
```

Git history is here for anyone who cares.

Reply:
It is easier to keep it like that for major changes

Reply:
Given the notable scope extension, at least a quick note is appropriate IMO.